packages/lwc-language-server/src/config.ts

```diff
@@ -15,11 +16,11 @@ export interface IJsconfig {
         paths?: IPaths;
     };
 }
-
+// this listens and i have to detect where this file is coming from
 export async function onIndexCustomComponents(context: WorkspaceContext, files: string[]) {
     // set paths for all current components in all the projects jsconfig.json files
     for (const relativeModulesDir of await context.getRelativeModulesDirs()) {
-        const modulesDir = path.join(context.workspaceRoot, relativeModulesDir);
+        const modulesDir = path.join(context.workspaceRoots[0], relativeModulesDir);
```

Comment:
Logic in this method looks wrong to me now - I think we need to get rid of this getRelativeModulesDir() method and just have that method return the full list of absolute paths to module directories, rather than getting relative ones and having to do path joins. That will simplify this greatly.